integration-tests/src/test/java/eu/tsystems/mms/tic/testframework/test/utils/FileDownloaderTest.java

```diff
@@ -97,29 +101,28 @@ public void testT02_downloadFileOfHttpsUrl() throws IOException {
     public void test03_downloadFileToLongLocation() throws IOException {
 
         final WebDriver driver = WebDriverManager.getWebDriver();
-        FileDownloader downloader = new FileDownloader(FileUtils.getUserDirectoryPath() + "/foo/bar\\test", true, true).setProxy(null);
+        FileDownloader downloader = new FileDownloader(FileUtils.getUserDirectoryPath() + "/foo/bar\\test", false, false).setProxy(null);
+        downloader.setConnectionConfigurator(createConnectionConfigurator());
 
-        String download = downloader.download(driver, WebDriverManager.getWebDriver().getCurrentUrl(),
-                "test03_downloadFileToLongLocation.htm");
-        File file = FileUtils.getFile(download);
+        File file = downloader.download(driver.getCurrentUrl(), "test03_downloadFileToLongLocation.htm");
 
         Assert.assertTrue(file.exists(), "File was downloaded correctly.");
     }
 
     @Test
     public void test04_readFileNameFromResponseHeader() throws IOException {
-        WebDriver driver = WebDriverManager.getWebDriver();
         FileDownloader downloader = new FileDownloader();
-        File file = downloader.download(driver, "https://upload.wikimedia.org/wikipedia/de/thumb/e/e1/Java-Logo.svg/800px-Java-Logo.svg.png");
-        Assert.assertEquals(file.getName(), "800px-Java-Logo.svg");
+        DefaultConnectionConfigurator connectionConfigurator = createConnectionConfigurator();
+        connectionConfigurator.setHeader("Accept", "text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9");
+        downloader.setConnectionConfigurator(connectionConfigurator);
+        File file = downloader.download( "https://images.pexels.com/users/avatars/616468/kira-schwarz-869.jpeg?auto=compress&fit=crop&h=256&w=256");
```

Comment:
The reason is here, the `Content-Disposition` header is only send by `images.pexels.com` when the `Accept` header for `image/webp` is available AND, much more important: the `User-Agent` is defined. Otherwise, the host will send an `503 Service Unavailable`.